lib/airbrake-ruby/sync_sender.rb

```diff
@@ -14,6 +14,7 @@ class SyncSender
     # @param [Airbrake::Config] config
     def initialize(config)
       @config = config
+      @rate_limit_reset = Time.now
```

Comment:
This can be initialized to `0`

Reply:
I wish it could work, but it won't. We can't compare Time and Integers:

```hs
[4] pry(main)> Time.now > 0
ArgumentError: comparison of Time with 0 failed
from (pry):4:in `>'
[5] pry(main)> 0 < Time.now
ArgumentError: comparison of Fixnum with Time failed
from (pry):5:in `<'
```